.circleci/config.yml

```
@@ -102,6 +102,11 @@ aliases:
 
 workflows:
   test_and_release:
+    when:
+      not:
+        matches:
+            pattern: /^l10n_crowdin_action$/
+            value: << pipeline.git.branch >>
```

Incorrect indentation
```suggestion
          pattern: /^l10n_crowdin_action$/
          value: << pipeline.git.branch >>
```